libosmscout-import/include/osmscout/import/GenCoordDat.h

```diff
@@ -29,6 +29,15 @@
 
 namespace osmscout {
 
+/**
+ * Some temporary variables used in asserts may be unused
+ * on production build, when asserts are wiped out.
+ *
+ * Usage of this macro for such variables makes production
+ * build happy, and avoids unused-variable warning
+ */
+#define _unused(x) ((void)(x))
```

Comment:
> ...more central location
I agree. We have more such warnings, where some arguments are unused just on windows... But where it should be? 
`osmscout/system/Assert.h`, `osmscout/system/Compiler.h` or somewhere else?

> Is underscore in macros allowed...
I have no idea :-) But underscore don't makes sense here, we can remove it...